worlds/messenger/__init__.py

```diff
@@ -82,7 +82,9 @@ def generate_early(self) -> None:
         self.shop_prices, self.figurine_prices = shuffle_shop_prices(self)
 
     def create_regions(self) -> None:
-        self.multiworld.regions += [MessengerRegion(reg_name, self) for reg_name in REGIONS]
+        for region in [MessengerRegion(reg_name, self) for reg_name in REGIONS]:
```

Comment:
This looked odd to me, as it seemed like it was creating and  discarding the regions. I then found  that this is implicit now, which could probably do with at least a comment here, if not making it explicit.